webapi/src/main/scala/org/knora/webapi/responders/v2/ResourcesResponderV2.scala

```diff
@@ -1274,6 +1274,24 @@ class ResourcesResponderV2(responderData: ResponderData) extends ResponderWithSt
                 }
             }
 
+            // get standoff link values
+            standoffLinkValuesOption: Option[Seq[ReadValueV2]] = resource.values.get(OntologyConstants.KnoraBase.HasStandoffLinkToValue.toSmartIri)
+
+            standoffLinkValues: Seq[ReadLinkValueV2] = standoffLinkValuesOption match {
+
+                case Some(valueSeq: Seq[ReadValueV2]) =>
```

Comment:
@benjamingeer In a first approach, I tried to match directly for `Seq[ReadLinkValueV2]` but I got a compiler warning: "non-variable type argument is unchecked since it is eliminated by erasure".

I found a [post](https://stackoverflow.com/questions/25222989/erasure-elimination-in-scala-non-variable-type-argument-is-unchecked-since-it) which discusses that issue and use `collect` now which works fine. 

Do you think that is a good solution?

Reply:
I am reading http://squidarth.com/scala/types/2019/01/11/type-erasure-scala.html now ...

Reply:
> got a compiler warning

Yes, I think this because you’re matching on a subtype of the element type in a collection. If `collect` does what you need, that’s great. Otherwise you could iterate over the elements and `match` each element.

Reply:
`collect` does the job :-)